RAW/ViewController.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  ViewController.swift
+//  RAW
+//
+//  Created by Bharat malhotra on 13/07/18.
+//  Copyright © 2018 Bharat malhotra. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+
+    @IBOutlet weak var tableView: UITableView!
+    var dataArr : [RawModel] = []
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view, typically from a nib.
+        fetchDataWithAPI()
+    }
+    
+    func fetchDataWithAPI(){
+        WebService().fetchRawData(sucess:{[weak self](userData) in
+           
+            DispatchQueue.main.async {[weak self] () -> Void in
+                guard let `self` = self else {return}
+                self.dataArr = userData
+                self.tableView.reloadData()
+            }
+            }, failure:{[weak self](error) in
+                guard let `self` = self else {return}
+                let alert = UIAlertController(title:"Alert", message:error, preferredStyle:.alert)
+                alert.addAction(UIAlertAction.init(title:"ok", style:.cancel, handler:nil))
+                self.present(alert, animated: true, completion: nil)
+        })
+    }
+    
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
```

Comment:
why do we have these methods when we are not using them?

Reply:
will remove them.